modules/p2-profile-gen/pom.xml

```diff
@@ -945,6 +951,17 @@
                                     <id>org.wso2.carbon.identity.developer.language-server.feature.group</id>
                                     <version>${org.wso2.carbon.identity.developer.version}</version>
                                 </feature>
+
+
+                                <feature>
+                                    <id>org.wso2.carbon.extension.identity.helper.feature.group</id>
+                                    <version>${identity.extension.utils}</version>
+                                </feature>
+                                <feature>
+                                    <id>org.wso2.carbon.extension.identity.authenticator.totp.connector.feature.group</id>
```

Comment:
let's remove the word 'connector' as it will no longer be a connector

Reply:
Tracked this via https://github.com/wso2/product-is/issues/11006